common/presenters/moves-by-location.js

```diff
@@ -12,7 +12,7 @@ module.exports = function movesByLocation(data, locationKey = 'to_location') {
     locations.push({
       sortKey: location,
       items: sortBy(moves, 'profile.person._fullname').map(
-        moveToCardComponent()
+        moveToCardComponent({ showIdentifierNumber: true })
```

Comment:
There's [another instance](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/f689f5b35c63585384896ad305891c4a7c2df411/common/presenters/moves-by-vehicle.js#L29) of this presenter being used when we display moves by vehicle. Can we update that to also include the identifier because I think it should follow the same behaviour.